### ISSUE TITLE
Org blocks uses green for unstyled syntax when language is set

### ISSUE THREAD

**Issue by Midnight1938** [catppuccin/emacs#61]
The syntax for C++ is not highlighted correctly, or has colours that are too similar

## Others
| purple | nord |
|--|--|
|<img src="https://user-images.githubusercontent.com/58230351/217450162-673b55f8-95d6-49f5-b4ac-676036b527a1.png" width="80%">|<img src="https://user-images.githubusercontent.com/58230351/217450243-81daf430-5ada-4cab-ad7a-2995dec73080.png" width ="80%">|

## Catppuccin
![2023-02-08T11:46:31,742772803+05:30](https://user-images.githubusercontent.com/58230351/217450049-0aedc8fe-8f1b-4223-8807-231b37030355.png)

**Comment by nyxkrage** [comment]
Hey 👋

I am actually aware of this, and it seems to be an issue with specifically org-mode source block, that use the normal source block face rather than `default` for unstyled parts.

I am looking into it, but haven’t found a solution nor workaround yet.

**Comment by nyxkrage** [comment]
Yeah so all the other themes have the same problem, what is happening with Catppuccin is that both string and the source blocks are green. What purple and nord have is that their source blocks without a language set are just white. As a workaround you can change the color of source blocks to text with

```elisp
(defun ctp/text-org-blocks ()
   (face-remap-add-relative 'org-block (list :foreground (catppuccin-get-color 'text))))

(add-hook! 'org-mode-hook 'ctp/text-org-blocks)
```

I'll leave the issue open until a proper solution is found.

**Comment by nyxkrage** [comment]
This should now be possible to overwrite with org-mode 9.6.4 with
```lisp
(setq org-src-fontify-natively t)
(add-to-list 'org-src-block-faces '("" '(:foreground (catppuccin-get-color 'text))))
```

I will keep this open until I get around to making a PR that has catppuccin automatically set this.

ref: [org-mode: Fontify src blocks without lang](https://git.savannah.gnu.org/cgit/emacs/org-mode.git/commit/?id=c2ffc6b60)

**Comment by nyxkrage** [comment]
org-mode 9.7 has since been released
